### ISSUE TITLE
Expand Orbot/Tor mode

### ISSUE THREAD

**Issue by n8fr8** [anthonycr/Lightning-Browser#244]
We are currently working on our EOL plans for Orweb, with our new focus on Orfox, a Firefox-based/Tor-Browser based browser (https://dev.guardianproject.info/projects/orfox-private-browser/).

I already have started promoting Lightning more to users who want something more powerful than Orweb, but I have also been thinking that not everyone is going to want to use Orfox, since it is going to be very memory/storage intensive. We also need some sort of upgrade path for our 1M+ Orweb users that isn't jarring.

I have two ideas that I have been considering:

1) Work with you to create an Orweb-branded version of Lightning, that would have different default settings aimed to be more private, secure... this might include having javascript off by default, setting DDG as the default search engine, clear cache on exit, always use incognito, etc....

2) Add a more complete "private mode" or "super/actual incognito" feature in Lightning, that would activate when you detect Orbot... "It looks like you have Orbot installed, do you want to enable the super-incognito mode?"

Any thoughts, preferences?

**Comment by n8fr8** [comment]
(btw, Lightning is my default browser now, especially on my Moto E/G devices, where Chrome is a complete memory hog... keep up the great work!)

**Comment by str4d** [comment]
+1 for this effort. I also use Lightning for my Orbot web browsing, and particularly appreciate the recently-added setting to enable Orbot outside the post-install dialog.

Unfortunately Lightning doesn't have support yet for [I2P Android](https://github.com/i2p/i2p.android.base), which Orweb does (because it just exposes an HTTP proxy port setting), but I am going to pull down the Lightning source and see what I can do :-)

**Comment by str4d** [comment]
#249 ready for comment :-)

**Comment by anthonycr** [comment]
I will give this some thought, it seems like a great idea to me. I kinda need to solidify a direction to head with Lightning because it's getting to a point where the features needed to keep the app competitive are out of my scope right now (stuff like across-device sync), so a further focus on privacy would be a good way for me keep the browser small without development stagnating.

@str4d I'll look over your PR, thanks :)

**Comment by n8fr8** [comment]
Yeah, you already have ad-blocking, flash disable, and the various clear cache on exit, and so on. I think a privacy focus, plus lightweight, small, quick, etc would be great. Chrome has gotten really bloated especially on my Moto's.

**Comment by afwang** [comment]
I'd like to recommend an option to use Ixquick instead of (or in addition to) DDG, unless some major reason that I overlooked would prevent Ixquick to be a search engine option.

I definitely love how well Lightning works on my device with Orbot instead of Orfox (~90 MiB vs. ~120 MiB makes a world of difference).
